feature-libs/product-configurator/rulebased/components/previous-next-buttons/configurator-previous-next-buttons.component.spec.ts

```diff
@@ -294,4 +309,28 @@ describe('ConfigPreviousNextButtonsComponent', () => {
 
     expect(configurationGroupsService.navigateToGroup).toHaveBeenCalledTimes(1);
   });
+
+  it('should call focusFirstAttribute', () => {
+    //usage of TestScheduler because of the async check in last line
+    const testScheduler = new TestScheduler((actual, expected) => {
+      expect(actual).toEqual(expected);
+    });
+    testScheduler.run(() => {
```

Comment:
Could we do this using a done() method, like e.g. in  /feature-libs/product-configurator/rulebased/root/cpq/interceptor/cpq-configurator-rest.interceptor.spec.ts